packages/connectivity/src/scp-cf/destination/destination-service.ts

```diff
@@ -123,9 +106,9 @@ async function fetchDestinations(
     headers
   )
     .then(response => {
-      const destinations: Destination[] = response.data
-        .filter(isParsable)
-        .map(parsableDestination => parseDestination(parsableDestination));
+      const destinations: Destination[] = response.data.map(
+        parsableDestination => parseDestination(parsableDestination)
```

Comment:
```suggestion
        destination => parseDestination(destination)
```
[nth] `isParsable` is not used to filter so this should be named accordingly.